requirements.txt

```diff
@@ -0,0 +1,8 @@
+opencv-python
+tqdm
+numpy
+argparse
+imgaug
+keras
+tensorflow
+h5py
```

Comment:
New line plz?

Reply:
requirements.txt files aren't necessarily a part of Python, but they are part of pip. (Python's package manager). With requirement files, txt is the standard format. Also, I can add a newline if you want.